cmd/shell.go

```diff
@@ -260,10 +260,47 @@ type CAConfig struct {
 }
 
 // PAConfig specifies how a policy authority should connect to its
-// database, and what policies it should enforce.
+// database, what policies it should enforce, and what challenges
+// it should offer.
 type PAConfig struct {
 	DBConnect              string
 	EnforcePolicyWhitelist bool
+	Challenges             map[string]bool
+}
+
+// UnmarshalJSON is really actually vanilla, but with some validity checks and
```

Comment:
Could you clarify why you think it's better to have the vanilla unmarshal?  It seems like by not doing the validation/default setting all the time, you're inviting cases where invalid configs get accepted.